src/lib/logical_query_plan/lqp_utils.hpp

```diff
@@ -100,7 +100,7 @@ enum class LQPVisitation { VisitInputs, DoNotVisitInputs };
 /**
  * Calls the passed @param visitor on @param lqp and recursively on its INPUTS. This will NOT visit subqueries.
  * The visitor returns `ExpressionVisitation`, indicating whether the current nodes's input should be visited
- * as well.
+ * as well. The algorithm is BFS.
```

Comment:
ASS

Reply:
What is the full name of this algorithm?

Reply:
breadth-first search

Reply:
What does `ASS` mean?

Reply:
[ASS](https://gist.github.com/klaaspieter/12cd68f54bb71a3940eae5cdd4ea1764) ;)

Reply:
Alright, now I don't have to mess up my search history anymore. D: